eodatasets3/images.py

```diff
@@ -814,6 +814,79 @@ def create_thumbnail(
                                 index,
                             )
 
+    def create_thumbnail_from_numpy(
+        self,
+        rgb: Tuple[numpy.array, numpy.array, numpy.array],
+        out_scale=10,
+        resampling=Resampling.average,
+        static_stretch: Tuple[int, int] = None,
+        percentile_stretch: Tuple[int, int] = (2, 98),
+        compress_quality: int = 85,
+        input_geobox: GridSpec = None,
+        nodata: int = -999,
+    ):
+        """
+        Generate a numpy which can save as jpg image using the given three arrays as red, green, blue.
```

Comment:
Make it slightly clearer why we're duplicating the function
```suggestion
        Generate a thumbnail as numpy arrays.
        
        Unlike the default `create_thumbnail` function, this is done entirely in-memory. It will likely require more
        memory but does not touch the filesystem.
```

Reply:
Thanks a lot for the text.